compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -11465,43 +11465,45 @@ OMR::Z::TreeEvaluator::loadaddrEvaluator(TR::Node * node, TR::CodeGenerator * cg
    if (symRef->isUnresolved())
       {
 #ifdef J9_PROJECT_SPECIFIC
-      if (cg->getS390ProcessorInfo()->supportsArch(TR_S390ProcessorInfo::TR_z10))
-         {
-         TR::UnresolvedDataSnippet * uds;
-         TR::Instruction * cursor;
+      // create UnresolvedDataSnippet
```

Comment:
This comment seems not useful. We can remove it.

Reply:
Fixed.